locales/br.json

```
@@ -206,6 +212,15 @@
   "Settings_MagicPlus_Enable": "Activer Magic+",
   "Settings_MagicPlus_Description_Optimization": "Optimise automatiquement l’organisation de tes tâches pour améliorer ta productivité",
   "Settings_MagicPlus_ExplanationLocal": "Modèle entièrement local, aucune donnée n’est transférée en dehors de ton appareil",
+  "Settings_MagicPlus_Unloading": "Déchargement de Magic+",
```

```suggestion
  "Settings_MagicPlus_Unloading": "Désactivation de Magic+",
```
"désactivation" serait peut-être plus approprié que "déchargement" ?